### PR TITLE
Fix typos in comments

### PR DESCRIPTION
**Pull request by yamt**

### CODE REVIEW

File: include/nuttx/fs/fs.h [hunk 1]
```diff
@@ -179,7 +179,8 @@
  * the open method will redirect to another driver and return a file
  * descriptor instead.
  *
- * This case is when SUSv1 pseudo-terminals are used (CONFIG_PSEUDOTERM_SUSV1=y).
+ * This case is when SUSv1 pseudo-terminals are usedi
```
Comment (xiaoxiang781216) [inline include/nuttx/fs/fs.h:182]: usedi->used?

Comment (patacongo) [reply include/nuttx/fs/fs.h:182]: That happens when I use vim.  i = INSERT mode.

Comment (patacongo) [reply include/nuttx/fs/fs.h:182]: I will fix that.

Comment (patacongo) [reply include/nuttx/fs/fs.h:182]: See PR #778

Comment (yamt) [reply include/nuttx/fs/fs.h:182]: oops, sorry.
yes, i'm using vi. and my "i" key has a bad chattering. i plan to fix the keyboard.

**Pull request merged**